_includes/icons.html

```diff
@@ -1,158 +1,158 @@
+    <a class="no-link-style" href="http://www.pinterest.com/{{ include.data.pinterest }}" title="{% translate global.follow_us_on %} Pinterest">
         <i class="fa fa-fw fa-pinterest"></i>
     </a>
 </li>
 {% endif %}
 
 {% if include.data.reddit %}
 <li>
-    <a class="no-link-style" href="https://www.reddit.com/user/{{ include.data.reddit }}" title="{{ include.data.str_follow_on }} Reddit">
+    <a class="no-link-style" href="https://www.reddit.com/user/{{ include.data.reddit }}" title="{% translate global.follow_us_on %} Reddit">
         <i class="fa fa-fw fa-reddit"></i>
     </a>
 </li>
 {% endif %}
 
 {% if include.data.soundcloud %}
 <li>
-    <a class="no-link-style" href="https://soundcloud.com/{{ include.data.soundcloud }}" title="{{ include.data.str_follow_on }} SoundCloud">
+    <a class="no-link-style" href="https://soundcloud.com/{{ include.data.soundcloud }}" title="{% translate global.follow_us_on %} SoundCloud">
         <i class="fa fa-fw fa-soundcloud"></i>
     </a>
 </li>
 {% endif %}
 
 {% if include.data.stack_exchange %}
 <li>
-    <a class="no-link-style" href="{{ include.data.stack_exchange }}" title="{{ include.data.str_follow_on }} Stack Exchange">
+    <a class="no-link-style" href="{{ include.data.stack_exchange }}" title="{% translate global.follow_us_on %} Stack Exchange">
         <i class="fa fa-fw fa-stack-exchange"></i>
     </a>
 </li>
 {% endif %}
 
 {% if include.data.steam %}
 <li>
-    <a class="no-link-style" href="http://steamcommunity.com/id/{{ include.data.steam }}" title="{{ include.data.str_follow_on }} Steam">
+    <a class="no-link-style" href="http://steamcommunity.com/id/{{ include.data.steam }}" title="{% translate global.follow_us_on %} Steam">
         <i class="fa fa-fw fa-steam"></i>
     </a>
 </li>
 {% endif %}
 
 {% if include.data.tumblr %}
 <li>
-    <a class="no-link-style" href="http://{{ include.data.tumblr }}.tumblr.com/" title="{{ include.data.str_follow_on }} Tumblr">
+    <a class="no-link-style" href="http://{{ include.data.tumblr }}.tumblr.com/" title="{% translate global.follow_us_on %} Tumblr">
         <i class="fa fa-fw fa-tumblr"></i>
     </a>
 </li>
 {% endif %}
 
 {% if include.data.wordpress %}
 <li>
-    <a class="no-link-style" href="https://{{ include.data.wordpress }}.wordpress.com/" title="{{ include.data.str_follow_on }} WordPress">
+    <a class="no-link-style" href="https://{{ include.data.wordpress }}.wordpress.com/" title="{% translate global.follow_us_on %} WordPress">
         <i class="fa fa-fw fa-wordpress"></i>
     </a>
 </li>
 {% endif %}
 
 {% if include.data.youtube %}
 <li>
-    <a class="no-link-style" href="https://www.youtube.com/channel/{{ include.data.youtube }}" title="{{ include.data.str_follow_on }} YouTube">
+    <a class="no-link-style" href="https://www.youtube.com/channel/{{ include.data.youtube }}" title="{% translate global.follow_us_on %} YouTube">
         <i class="fa fa-fw fa-youtube"></i>
     </a>
 </li>
```

Comment:
You forget to update corporate logo.
```js
{% if include.data.corporate %}
<li>
    <a class="no-link-style" href="{{ include.data.corporate }}" title="{{ translate global.corporate }}">
        <img class="corporate-logo" src="{{ include.data.corporate_logo | prepend: site.baseurl }}" alt="corporate_logo" width="24.42" height="19" />
    </a>
</li>
{% endif %}
```
When switching to french, logo is not found anymore...

Reply:
I found why there is a problem with the logo. You must replace `site.baseurl` by `site.baseurl_root`.